### PR TITLE
chore: add constrain formatter

### PR DESCRIPTION
**Pull request by f01dab1e**
# Description

# PR Checklist

- [x] I have tested the changes locally.
- [x] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: tooling/nargo_fmt/src/visitor/stmt.rs [hunk 1]
```diff
@@ -28,8 +28,36 @@ impl super::FmtVisitor<'_> {
 
                     self.push_rewrite(format!("{let_str} {expr_str};"), span);
                 }
+                StatementKind::Constrain(constrain) => {
+                    let ConstrainStatement(expr, message, kind) = constrain;
```
Comment (jfecher) [inline tooling/nargo_fmt/src/visitor/stmt.rs:32]:
```suggestion
                StatementKind::Constrain(ConstrainStatement(expr, message, kind)) => {
```

**Review by jfecher — APPROVED** [review]
👍

**Comment by f01dab1e** [comment]
@jfecher

**Review by kevaundray — APPROVED** [review]
approving by proxy of Jake

**Pull request merged**